module/surveys/helper.js

```diff
@@ -134,11 +134,6 @@ module.exports = class SurveysHelper {
         newSolutionDocument.isReusable = true;
         newSolutionDocument.status = messageConstants.common.ACTIVE_STATUS;
         newSolutionDocument.isExternalProgram = solutionData?.isExternalProgram ?? false;
-        let date = new Date();
-        newSolutionDocument.startDate = solutionData.startDate ? new Date(solutionData.startDate) : new Date();
```

Comment:
@borkarsaish65 if solutionData.startDate  then add  newSolutionDocument.startDate=solutionData.startDate
only else part we are avoiding right?

Reply:
No